workos/events.py

```diff
@@ -78,4 +121,11 @@ def list_events(
             "method": Events.list_events,
         }
 
+        data = []
+        for list_data in response["data"]:
+            if list_data["event"] in EVENT_TO_OBJECT.keys():
```

Comment:
if we add a new event or update an event type, until the SDK is explicitly updated we'll just ignore it